Assets/MRTK/Core/Providers/Hands/BaseHandVisualizer.cs

```diff
@@ -126,6 +126,11 @@ public virtual void OnHandJointsUpdated(InputEventData<IDictionary<TrackedHandJo
                 for (int i = 1; i < ArticulatedHandPose.JointCount; i++)
                 {
                     TrackedHandJoint handJoint = (TrackedHandJoint)i;
+                    // Skip this hand joint if the event data doesn't have an entry for it
+                    if (!eventData.InputData.ContainsKey(handJoint))
```

Comment:
Could we use TryGetValue here instead of an explicit Contains + indexing?

Reply:
Handled in a followup PR